util/helpers.py

```diff
@@ -0,0 +1,12 @@
+def check_cancel(lst):
+    """Check if the user has cancelled the operation.
+
+    Args:
+        lst (any): User's response in list format
+    returns:
+        bool: True if cancelled, False otherwise
+    """
+    for word in lst:
```

Comment:
`return "cancel" in lst`

Reply:
doesn't work
"Cancel", "CANCEL" or any of its variants might be present in lst